kong/plugins/datadog/handler.lua

```diff
@@ -28,10 +28,11 @@ local get_consumer_id = {
 }
 
 
-local function compose_tags(service_name, status, consumer_id, tags, conf)
+local function compose_tags(service_name, status, consumer_id, tags, conf, route_name)
   local result = {
     (conf.service_name_tag or "name") .. ":" .. service_name,
-    (conf.status_tag or "status") .. ":" .. status
+    (conf.status_tag or "status") .. ":" .. status,
+    route_name and route_name ~= "" and ((conf.route_name_tag or "route") .. ":" .. route_name)
```

Comment:
I think the checks `route_name and route_name ~= ""` can be omitted right? We know it'll either have a value or be `""`, and in case it's `""` we probably want to just use it like that, or else this evaluation will return `false`, am I right?